src/Platforms/MySQLPlatform.php

```diff
@@ -667,14 +668,12 @@ protected function getPreAlterTableIndexForeignKeySQL(TableDiff $diff)
             $diff->removedForeignKeys = [];
         }
 
-        $sql = array_merge(
+        return array_merge(
             $sql,
             $this->getPreAlterTableAlterIndexForeignKeySQL($diff),
             parent::getPreAlterTableIndexForeignKeySQL($diff),
             $this->getPreAlterTableRenameIndexForeignKeySQL($diff)
         );
```

Comment:
If we want to spare us the `array_merge()` call in line 657, we can do this:

```suggestion
        return array_merge(
            [],
            ...$sql,
            ...[
                $this->getPreAlterTableAlterIndexForeignKeySQL($diff),
                parent::getPreAlterTableIndexForeignKeySQL($diff),
                $this->getPreAlterTableRenameIndexForeignKeySQL($diff),
            ]
        );
```